README.md

```diff
@@ -70,6 +70,18 @@ Genny has self-tests using Catch2. You can run them with the following command:
 ./scripts/lamp cmake-test
 ```
 
+For more fine-tuned testing (eg. running a single test or excluding some) you
+can manually invoke the test binaries:
+
+```sh
+# Build genny first: `./scripts/lamp [...]`
+./build/src/gennylib/gennylib_test "My testcase"
+```
+
+Read more about what parameters you can pass [here][s].
+
+[s]: https://github.com/catchorg/Catch2/blob/master/docs/command-line.md#specifying-which-tests-to-run
```

Comment:
suggest `s/master/v2.5.0` in the URL in case they add parameters. So far there's no difference between the version we're using and master.

Reply:
Can we also name the link "catch2" instead of "s"? 🙂

Reply:
Done